applications/nrf5340_audio/src/audio/le_audio_rx.c

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <nrfx_clock.h>
+
+#include "nrf5340_audio_common.h"
+#include "streamctrl.h"
+#include "audio_datapath.h"
+#include "macros_common.h"
+#include "audio_system.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(le_audio_rx, CONFIG_LE_AUDIO_RX_LOG_LEVEL);
+
+static bool initialized;
+static struct k_thread audio_datapath_thread_data;
+static k_tid_t audio_datapath_thread_id;
+K_THREAD_STACK_DEFINE(audio_datapath_thread_stack, CONFIG_AUDIO_DATAPATH_STACK_SIZE);
+
+struct ble_iso_data {
```

Comment:
IMO the struct defines should be directly below #include (or below #define if there are #defines)